specs.json

```diff
@@ -456,6 +457,16 @@
       "llmOptions"
     ]
   },
+  "mcpServer": {
+    "properties": {
+      "url": "string",
+      "auth": "object",
+      "roots": ["#root"]
```

Comment:
What is root and where we define this root?

Reply:
https://spec.modelcontextprotocol.io/specification/2024-11-05/client/roots/